projects/valtimo/form-management/src/lib/form-management-routing.module.ts

```diff
@@ -47,6 +47,9 @@ const routes: Routes = [
 @NgModule({
   imports: [RouterModule.forChild(routes)],
   exports: [RouterModule],
-  declarations: [],
 })
-export class FormManagementRoutingModule {}
+export class FormManagementRoutingModule {
+  constructor(private router: Router) {
+    RouterUtils.configureNavigationCanceling(this.router);
```

Comment:
Why make configureNavigationCanceling a separate function? Why not a generic function like configureRouter which now configures that one property. In that case, we just need to edit that function and not adjust all router modules in the future if we want to configure another property by default